docs/protos/python.md

```diff
@@ -0,0 +1,394 @@
+
+    access_token = '<neosync-api-token>'
+    metadata = [('authorization', f'Bearer {access_token}')] if access_token else None
+
+    try:
+            # Make the RPC call
+            response = stub.AnonymizeSingle(
+                AnonymizeSingleRequest(
+                    input_data=input_data,
+                    transformer_mappings=[transformer_mapping],
+                    account_id="<neosync-api-url>"
+                ),
+                metadata=metadata
+            )
+
+            # Parse and print response
+            try:
+                output_data = json.loads(response.output_data)
+                print("Anonymized data:", output_data)
+            except json.JSONDecodeError:
+                print("Raw response:", response.output_data)
+    except grpc.RpcError as e:
+        if e.code() == grpc.StatusCode.UNAUTHENTICATED:
+            print("Authentication failed - please check your access token")
+        elif e.code() == grpc.StatusCode.UNAVAILABLE:
+            print("Failed to connect to server. Please check the hostname and port.")
+        else:
+            print(f"RPC failed: {e.code()}, {e.details()}")
+    finally:
+        channel.close()
+if __name__ == "__main__":
+    main()
+```
+
+Let's take a closer look at what we're doing here. Neosync's AnonymizeSingle API uses [JQ](https://jqlang.github.io/jq/manual/) expressions to target field(s) in your object. This means that you don't have to parse your object before sending it to Neosync. You can pass it in as-is and just write JQ expressions to target the field(s) that you want to anonymize or generate.
+
+Our output will look something like this:
+
+```js
+// output
+{'text': 'Dear Mr. <REDACTED>, your physical therapy for your rotator cuff injury is approved for 12 sessions. Your first appointment with therapist <REDACTED> is on <REDACTED> at <REDACTED>. Please bring a photo ID. We have your SSN on file as <REDACTED>. Is this correct?'}
+```
+
+As you can see, we've identified and redacted the PII in the original message and output a string that no longer contains PII. Alternatively, you can choose to Replace, Mask or even Hash the detected PII value instead of Redacting it.
+
+### Triggering a Job Run
+
+Another common use case is to create resources in Neosync such as Jobs, Connections, Runs, Transformers and more. In this example, we'll trigger a Job which will create a Job Run. This can be used as part of a set-up script or custom workflow. Let's take a look at the code:
+
+Let's augment our code from above to call the `CreateJobRun` API.
+
+```python
+import grpc
+import json
+
+from neosync.mgmt.v1alpha1.job_pb2_grpc import JobServiceStub
+from neosync.mgmt.v1alpha1.job_pb2 import CreateJobRunRequest
+
+from google.protobuf.json_format import MessageToDict, MessageToJson
+
+def main():
+
+    channel_credentials = grpc.ssl_channel_credentials()
+
+    channel = grpc.secure_channel("neosync-api.svcs.neosync.dev", channel_credentials)
```

Comment:
def a minor thing but I think in the other language tutorial you pull this from an environment variable.